src/test/java/com/emc/mongoose/storage/driver/pravega/integration/PravegaEventReadTest.java

```diff
@@ -35,7 +36,8 @@ public void testEventRead()
 						.build();
 		streamManager.createStream(scopeName, streamName, streamConfig);
 		try (
-						val clientFactory = ClientFactory.withScope(scopeName, controllerURI);
+						val clientFactory = EventStreamClientFactory.withScope(scopeName,
+								ClientConfig.builder().controllerURI(controllerURI).build());
```

Comment:
Shouldn't parameters listed from new line be at least 1 tab away from instance / class name? Like it does here: https://github.com/emc-mongoose/mongoose-storage-driver-pravega/blob/master/src/main/java/com/emc/mongoose/storage/driver/pravega/PravegaStorageDriver.java#L532